ipaserver/install/ldapupdate.py

```diff
@@ -903,13 +907,20 @@ def create_connection(self):
             raise RuntimeError("Offline updates are not supported.")
 
     def _run_updates(self, all_updates):
+        reindex_attributes = set()
```

Comment:
nit but I'm pretty sure updates only add indices so index_attributes is probably a clearer name.

Can you add a comment block somewhere about how the values are batched up, like you have in the commit message?

Reply:
I think updates can also modify an index (e.g. I'm adding a ``pres`` index to an existing index with ``eq``).

Sure, I'll add a comment.